conf/messages.en

```diff
@@ -179,6 +179,11 @@ containerIdentificationNumber.error.length = The identification number must be 1
 containerIdentificationNumber.error.characters = The identification number must only include letters a to z and numbers 0 to 9
 containerIdentificationNumber.error.duplicate = This identification number has already been submitted. Enter a unique container identification number
 
+transportEquipment.index.addSealYesNo.title = Do you want to add a seal for the transport equipment?
+transportEquipment.index.addSealYesNo.heading = Do you want to add a seal for the transport equipment?
+transportEquipment.index.addSealYesNo.paragraph = You can only add a seal if you are authorised to use customs seals.
+transportEquipment.index.addSealYesNo.error.required = Select yes if you want to add a seal for the transport equipment?
```

Comment:
Question mark presumably not needed in this one?

Reply:
Yea looks like a typo on the content doc and I've just copied and pasted, my bad!